### PR TITLE
Feature/accessibility

### PR DESCRIPTION
**Pull request by BHaddad1**
## What does this merge do? Check all that apply.
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## How has this been tested?
User tested on the site.

## Description:
This branch adds accessibility features in the HTML and scripts file.

## Issues:
No.

## Where to find Changes:
Files: HTML, CSS, Scripts.js.
Methods:
line Numbers if possible: 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -45,9 +45,9 @@ <h2 class='post-failure-display hidden' id="post-failure">Sorry, failed to post!
               <form>
                 <label class='hidden' for="calendar"> Calendar:</label>
                 <input type="date" id="calendar" name="calendar" required>
-                <button type="submit" id="dateInput" disabled> submit </button> 
+                <button type="submit" id="dateInput" tabindex="0">Submit</button>
```
Comment (AndrewCerveny) [inline dist/index.html:48]: if it's an html 5 ( button, main etc. tab index is part of the inbuilt functionality.

**Review by KarimAl-Rashdan — APPROVED** [review]
This looks good. site still functions as expected.

File: src/css/styles.css [hunk 2]
```diff
@@ -210,6 +209,16 @@ p {
   width: 25%;
 }
 
+.sleep:focus,
+.hydration:focus,
+.step-goal:focus,
+.activity:focus,
+.return-to-widget:focus,
+.input-btn:focus {
```
Comment (AndrewCerveny) [inline src/css/styles.css:217]: well done nice stack!

**Review by AndrewCerveny — APPROVED** [review]
Went through it line- by - line looks good

**Pull request merged**